### PR TITLE
Run cpplint in quiet mode

### PR DESCRIPTION
**Pull request by areusch**
Removes lots of
```
Done processing ...
```

spam from cpplint logs.

cc @driazati

### CODE REVIEW

**Comment by areusch** [comment]
also cc @tqchen due to related dmlc-core change.

File: tests/lint/cpplint.sh [hunk 1]
```diff
@@ -17,8 +17,9 @@
 # under the License.
 
 
-python3 3rdparty/dmlc-core/scripts/lint.py vta cpp vta/include vta/src
-python3 3rdparty/dmlc-core/scripts/lint.py tvm cpp \
+echo "Running 2 cpplints (VTA and TVM)..."
+python3 3rdparty/dmlc-core/scripts/lint.py --quiet vta cpp vta/include vta/src
+python3 3rdparty/dmlc-core/scripts/lint.py --quiet tvm cpp \
```
Comment (driazati) [inline tests/lint/cpplint.sh:22]: These take kind of a while so it'd be nice to mention them separately

```suggestion
echo "Running VTA cpplint"
python3 3rdparty/dmlc-core/scripts/lint.py --quiet vta cpp vta/include vta/src
echo "Running TVM cpplint"
python3 3rdparty/dmlc-core/scripts/lint.py --quiet tvm cpp \
```

**Review by driazati — APPROVED** [review]

**Comment by areusch** [comment]
:/ i think maybe the rust build is busted when dlpack is updated.

**Comment by driazati** [comment]
#10897 also gets rid of the output

**Review by tqchen — APPROVED** [review]

**Comment by areusch** [comment]
blocked on https://github.com/dmlc/dmlc-core/pull/658#pullrequestreview-949205382

**Comment by areusch** [comment]
busted copy constructor was reverted for now, retrying this one

**Pull request merged**